whetstone/src/main/java/com/deliveryhero/whetstone/Whetstone.kt

```diff
@@ -105,12 +107,16 @@ public object Whetstone {
      * @see [installFragmentFactory]
      */
     public fun inject(activity: FragmentActivity) {
+        GlobalAndroidComponentListener.componentInjectionListener
```

Comment:
do we really need a separate object to hold this `componentInjectionListener`? what about having it right here, in `Whetstone`?

Reply:
hmm I was thinking about something for logging like okhttp that can be enabled for other parts later and I wanted all the related classes be in one package. on the other hand I didn't want to do this now and it was the insight behind having another object to be more extendable later.

Reply:
makes total sense, thank you. but maybe `logging` package in this case is not the best name? because currently it doesn't log anything. maybe `event` or `listener` suits better this use-case?